### ISSUE TITLE
About Hyperparmeters

### ISSUE THREAD

**Issue by Syzseisus** [GRAPH-0/RGTN-NIE#10]
Hi, thanks you for your wonderful work!

~In your paper GENI, you wrote "The dimension of predicate embedding was set to 10 for all KGs," in Appendix B.3
But I failed to find the specific code relate to this.
Would you let me know where it is or how to do?~
Sorry. I was confused.

My issues are:
1. Is it fair to use same `learning rate` and `weight decay` for all KGs?
FB15k has about 15k, and IMDB_S has about 1.1M nodes.
So it might be unfair to use same `learning rate` and `weight decay`.

2. There are so many hyperparameters
I can find a lots of hyperparameters but never used: `residual`, `attn-drop`
And those only used  the default: `num-heads`, `num-out-heads`, `num-layers`, `num-hidden`, `in-drop`, `negative-slope`, `pred-dim`.
Why did you designate them using argparse?

Sincerely,
Syzseisus

**Comment by GRAPH-0** [comment]
Hi Syzseisus,

1. Yes, you are free to search `lr` and `wd` for better performance on different KGs. I didn't sweep these hyperparameters before.
2. These hyperparameters may have been used during development but not cleaned up. Similarly, you can adjust these default hyperparameters based on your needs. If they are not needed, they can be removed from argparse.

Sincerely,
Han

**Comment by Syzseisus** [comment]
Thanks a lot
